packages/button/__tests__/Button.test.tsx

```diff
@@ -4,9 +4,11 @@ import { axe } from 'jest-axe'
 import type { ButtonProps } from '../src/'
 import Button from '../src/'
 
+const children = 'children'
+
 describe('accessibility', () => {
   it('is accessible with text', async () => {
-    const { container } = render(<Button>text</Button>)
+    const { container } = render(<Button>{children}</Button>)
     expect(await axe(container)).toHaveNoViolations()
   })
```

Comment:
Would it be meaningful to test accessibility for each of the `as` options?

Reply:
Good point, I'll add an accessibility test for button link

Reply:
a9e16a9